docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+services:
+  nfldb:
+    build:
+      context: .
+    depends_on:
+      - "postgres"
+    restart: always
+    command: ['./install/wait-for-postgres.sh', 'postgres', './scripts/nfldb-update', '--interval', '60' ]
```

Comment:
I personally think it would be a great idea to have that as a command. Mainly because when someone is able to start up their docker image, they will have the most up to date values for their nfldb